Sources/GRPCCore/Streaming/Internal/AsyncStream+MakeStream.swift

```diff
@@ -14,7 +14,8 @@
  * limitations under the License.
  */
 
-#if swift(<5.9)
+//#if swift(<5.9)
```

Comment:
The `#if` needs to stay -- `makeStream` was added in 5.9, this is a backport.

Reply:
Oops, I didn't mean to commit that, I was using it so I could see the call-sites